bedrock/settings/base.py

```diff
@@ -2216,3 +2216,15 @@ def facebook_tab_url_lazy():
 
     'DEFAULT_FILTER_BACKENDS': ('rna.filters.TimestampedFilterBackend',)
 }
+
+FIREFOX_OS_FEEDS = (
+    ('de', 'https://blog.mozilla.org/press-de/category/firefox-os/feed/'),
+    ('en-US', 'https://blog.mozilla.org/blog/category/firefox-os/feed/'),
+    ('es-ES', 'https://blog.mozilla.org/press-es/category/firefox-os/feed/'),
+    ('es', 'https://blog.mozilla.org/press-latam/category/firefox-os/feed/'),
+    ('fr', 'https://blog.mozilla.org/press-fr/category/firefox-os/feed/'),
+    ('it', 'https://blog.mozilla.org/press-it/category/firefox-os/feed/'),
+    ('pl', 'https://blog.mozilla.org/press-pl/category/firefox-os/feed/'),
+    ('pt-BR', 'https://blog.mozilla.org/press-br/category/firefox-os/feed/'),
+)
+FIREFOX_OS_FEED_LOCALES = [feed[0] for feed in FIREFOX_OS_FEEDS]
```

Comment:
scratch that, I was thinking of a previous implementation where the ordering *did* matter, but that's no longer the case. No reason we couldn't change it to a dict now.